admin/database/database.go

```diff
@@ -179,10 +185,18 @@ type DB interface {
 
 	FindProjectMemberUsers(ctx context.Context, projectID, afterEmail string, limit int) ([]*Member, error)
 	InsertProjectMemberUser(ctx context.Context, projectID, userID, roleID string) error
-	InsertProjectMemberUsergroup(ctx context.Context, groupID, projectID, roleID string) error
 	DeleteProjectMemberUser(ctx context.Context, projectID, userID string) error
 	DeleteAllProjectMemberUserForOrganization(ctx context.Context, orgID, userID string) error
 	UpdateProjectMemberUserRole(ctx context.Context, projectID, userID, roleID string) error
+	FindOrganizationUsergroups(ctx context.Context, orgID, afterName string, limit int) ([]*Usergroup, error)
+	FindAllUsergroupOrganizationRoles(ctx context.Context, orgID string) ([]*UsergroupOrgRole, error)
+	FindUsergroupOrganizationRole(ctx context.Context, groupID, orgID string) (*UsergroupOrgRole, error)
+	FindUsergroupAllProjectOrganizationRoles(ctx context.Context, orgID string) ([]*UsergroupProjectRole, error)
+	FindUsergroupProjectRoles(ctx context.Context, groupID, orgID string) ([]*UsergroupProjectRole, error)
+	UpsertOrganizationUsergroup(ctx context.Context, groupID, orgID, roleID string) error
+	DeleteOrganizationUsergroup(ctx context.Context, groupID, orgID string) error
+	UpsertProjectUsergroup(ctx context.Context, groupID, projectID, roleID string) error
+	DeleteProjectUsergroup(ctx context.Context, projectID, groupID string) error
```

Comment:
It would be nice if we could lean into the "member" terminology used for orgs and projects. But we have to be careful since a usergroup can both *have* members and *be* a member of something. I would propose using `MemberUsergroup` to consistently denote a "member that is a usergroup", and naming these functions `FindOrganizationMemberUsergroups`, `InsertOrganizationMemberUsergroup`, etc. Then they will align with the user-related functions `FindOrganizationMemberUser`, `InsertOrganizationMemberUser`, etc.